docs/contributors/document.md

```diff
@@ -0,0 +1,32 @@
+# Documentation Contributions
+
+Documentation for Gutenberg is maintained in the `/docs/` directory in the same Gutenberg Github repository. The docs are published every 15 minutes to the [Gutenberg Handbook site](https://wordpress.org/gutenberg/handbook/).
+
+## New Document
+
+To add a new documentation page:
+
+1. Create a Markdown file in the [docs](https://github.com/WordPress/gutenberg/tree/master/docs) folder
+2. Add item to the [toc.json](https://github.com/WordPress/gutenberg/blob/master/docs/toc.json) hierarchy
```

Comment:
For some reason I thought `toc.json` was being generated automatically now as part of `npm run docs:build`… am I wrong? I have clearly been tuned out 😓🙈

Reply:
I'm looking at the generator scripts that are called when running `docs:build` and I don't see where it writes toc.json. There is a section which writes out a table of contents, but that looks like it is only for the `Data Module Reference`
https://github.com/WordPress/gutenberg/blob/master/docs/tool/generator.js#L86

`manifest.json` is the only other file I found it generated, but their may be other processs or something I missed to. My method was searching for `writeFileSync` which is the JS function that writes out to a file.

Reply:
@mkaz you're right, this was my mistake 🤦‍♂️ toc.json is manually generated, it always has been and always will be!